diff --git a/src/llama-arch.cpp b/src/llama-arch.cpp
index a54bc19..ab907e9 100644
--- a/a/src/llama-arch.cpp
+++ b/b/src/llama-arch.cpp
@@ -2290,7 +2290,9 @@ static std::set<llm_tensor> llm_get_tensor_names(llm_arch arch) {
                 LLM_TENSOR_FFN_UP,
             };
         default:
-            GGML_ABORT("unknown architecture for tensor mapping");
+            return {
+                LLM_TENSOR_TOKEN_EMBD,
+            };
     }
 }
 
